### PR TITLE
Wrong menu text for adjusting Z probe offset

### PR DESCRIPTION
**Pull request by abortz**
Fix the text on the Z probe offset menu item in ProUI. The current one (recently changed) is verbose and overflows the space in the UI. The old one was perfect.

### CODE REVIEW

**Comment by classicrocker883** [comment]
I agree "Update Probe Z-Offset to " is a bit too wordy, but I would argue changing that to
```diff
- LSTR MSG_XATC_UPDATE_Z_OFFSET           = _UxGT("Update Probe Z-Offset to ");
+ LSTR MSG_XATC_UPDATE_Z_OFFSET           = _UxGT("Update Z-Offset");
```
or just make a new `MSG_`:
```diff
+ LSTR MSG_UPDATE_Z_OFFSET           = _UxGT("Update Z-Offset");
```

**Comment by abortz** [comment]
@classicrocker883 this is the same text used by a half dozen different UIs. Changing it seems like a topic for a different issue/PR.

@thinkyhead this seems like an easy revert -- the text string you switched to appears only to be used by an X Axis Twist wizard.

**Comment by sjasonsmith** [comment]
@abortz would you be able to link to the commit that changed the behavior you are reverting? If you could readily provide pictures of the before and after behavior that would be fantastic. Thanks.

**Comment by abortz** [comment]
Looks like it was 0f0955492994f4a7f4649c22e0f3aa5cc71541d7

Attached is an after photo, I don't have a before but the text was way too long and clipped.
PXL_20240407_155450101
